smart-contracts/contracts/mixins/MixinPurchase.sol

```diff
@@ -77,11 +76,10 @@ contract MixinPurchase is
     require(_recipient != address(0), 'INVALID_ADDRESS');
 
     // Let's get the actual price for the key from the Unlock smart contract
-    IUnlock unlock = IUnlock(unlockProtocol);
```

Comment:
These changes look ok to me. Originally, this line was using the full Unlock vs the interface. I changed it to use only the interface, and it looks like your changes extend this a bit to use the Unlock interface earlier on.

Reply:
Yup.  I read that best practices document yesterday and reacted to a couple suggestions like this.  Pretty minor change